Ludown/lib/parseFileContents.js

```diff
@@ -822,5 +792,31 @@ const addItemIfNotPresent = function(collection, type, value) {
         collection[type].push(itemObj);
     }  
 };
-
+/**
+ * Helper function to add an item to collection if it does not exist
+ * @param {object} collection contents of the current collection
+ * @param {LUISObjNameEnum} type item type
+ * @param {object} value value of the current item to examine and add
+ * @param {string []} roles possible roles to add to the item
+ * @returns {void} nothing
+ */
+const addItemOrRoleIfNotPresent = function(collection, type, value, roles) {
+    let existingItem = collection[type].filter(item => {return item.name == value});
```

Comment:
the braces and `return` statement can be omitted to produce the equivalent results:
```js
let existingItem = collection[type].filter(item => item.name == value);
```